contour/src/main/kotlin/com/squareup/contour/solvers/SimpleAxisSolver.kt

```diff
@@ -149,7 +178,6 @@ internal class SimpleAxisSolver(
         max = p0.resolve()
         mid = max - hV
         min = max - range
-        baseline = min + baselineRange
       }
     }
```

Comment:
I think we should also be able to add:
```
if (p0.point != Point.Baseline && baselineRange != Int.MIN_VALUE) {
  baseline = min + baselineRange
}
```
Think that can potentially save us an unnecessary `onMeasure` call

Reply:
I feel like `baseline()` already covers this. `measureSelf` is avoided if `baselineRange` is resolved. I could be wrong. Let me know, please.

Reply:
That is correct, but by moving it into resolveAxis, we could potentially resolve the baseline if we have perviously required layout.

```
view.topTo { 0 }

view1.topTo { view.baseline() } 
```

The above will trigger measure twice right now, because we need to measure `view` once to size it, and again to get the baseline. 

if we make the change above we'll grab the baseline on the first measure.

Reply:
I was meaning to build a tool to make this sort of analysis easier.